_episodes/03-loops.md

```diff
@@ -436,24 +430,57 @@ inflammation-12.csv
 ~~~
 {: .output}
 
-We've replaced `%d` in our earlier template with `%02d`.
-With this,
-we're specifying that we want our data to be displayed
-with a minimum width of 2 characters,
-and that we want to _pad_ with 0 for data
-that isn't at least 2 digits long.
+The final task is to generate the file names for the figures we're going to save.
+Let's name the output file after the data file used to generate the figure.
+So for the data set `inflammation-01.csv` we will call the figure `patient_data-01.png`.
```

Comment:
I must have lost track of this, but any reason not to call the output `inflammation-01.png` instead?

Reply:
I guess not! Just following the previous example.